action.yml

```diff
@@ -0,0 +1,87 @@
+name: 'send-notifications'
+author: '1up-team'
+description: 'Send build notifications to Slack channel'
+
+inputs:
+  slack-webhook-url:
+    description: 'Slack channel webhook URL to report to'
+    required: true
+
+  success-message:
+    description: "Slack message to send on success"
+    required: false
+
+  failure-message:
+    description: "Slack message to send on failure"
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - if: always() && github.ref == 'refs/heads/master'
+      shell: bash
+      run: |
+        GIT_COMMIT_MSG=$(git log -1 --pretty=format:'%s')
+        GIT_COMMIT_MSG_NO_SUFFIX="${GIT_COMMIT_MSG#*\(#}"
+        GIT_PR="${GIT_COMMIT_MSG_NO_SUFFIX%\)*}"
```

Comment:
Can we make this slightly more complicated so we can handle [1up-bff commits](https://github.com/spring-media/1up-bff/blob/c0f9c1d693fc1b0d1a388e3d23edaeb89f77ba03/.github/workflows/bff-api.yml#L178) as well?

Reply:
I've tried to merge with bff .. could you check again?